### DISCUSSION TITLE
posts/i-bought-the-stack-overflow-keyboard-i-instantly-became-a-10x-developer-and-ended-up-writing-the-default-firmware-for-it

### DISCUSSION THREAD

**Discussion by giscus** [teknologi-umum/blog#123] [General]
# posts/i-bought-the-stack-overflow-keyboard-i-instantly-became-a-10x-developer-and-ended-up-writing-the-default-firmware-for-it

https://teknologiumum.com/posts/i-bought-the-stack-overflow-keyboard-i-instantly-became-a-10x-developer-and-ended-up-writing-the-default-firmware-for-it

**Comment by lamualfa** [comment]
Hahaha 😆.

**Reply by KingPsychopath** [reply]
Great write up!
